README.md

```diff
@@ -15,3 +71,15 @@ A pre-commit hook will automatically lint and format staged `.js` files in the `
 
 To build the project, run:
 `npm run build`
+
+## Credits
+
+- [Utopia](https://utopia.fyi/)
+
+## License
+
+_EnChord_ is available under the MIT license. See the LICENSE file for more information.
+
+## Badges
+
+[![Netlify Status](https://api.netlify.com/api/v1/badges/61aa5294-7ef4-4d3d-978c-b61c9ac5fe4e/deploy-status)](https://app.netlify.com/sites/enchord/deploys)
```

Comment:
Instead of putting badges in their own section, we could also think about putting them closer to the top of the README, so people can easily get an overview of statuses for this project.

An example:
<img width="613" alt="Screenshot 2022-12-09 at 05 26 21" src="https://user-images.githubusercontent.com/62935115/206623925-c477706e-dc60-4a09-b991-ec40473c959b.png">

Reply:
That is indeed a better idea!